projects/dapp-wrose/description.yaml

```diff
@@ -14,8 +14,8 @@ license: Apache-2.0
 tags:
   - AuthZN
   - React
-codeUrl: https://github.com/oasisprotocol/dapp-wrose
+codeUrl: https://github.com/oasisprotocol/dapp-wrose/tree/ml/authzn
```

Comment:
This branch will be permanent?

Reply:
Repository is already archived, I would just leave it

Reply:
Oh. Github UI let me create a branch despite being archived :shrug: